bin/windows-toolset.ps1

```diff
@@ -171,3 +159,21 @@ cd $toolsDir\${opensslPkg}
 Invoke-External { & 7za x "$toolsDir\${opensslPkg}.tar" }
 
 cd $toolsDir
+# Download ruby
+Write-Host "Downloading http://buildsources.delivery.puppetlabs.net/windows/ruby/${rubyPkg}.7z"
```

Comment:
Can reuse the url like:

```powershell
$rubyUrl = "http://buildsources.delivery.puppetlabs.net/windows/ruby/$rubyPkg.7z"
Write-Host "Downloading $rubyUrl"
(New-Object net.webclient).DownloadFile($rubyUrl, "$toolsDir\${rubyPkg}.7z")
```

Seems like this would be the right spot to start consuming Ruby bits that are available in Nexus... but we're not publishing there yet.